src/components/App/App.js

```diff
@@ -1,13 +1,15 @@
 // import { Step, Stepper, StepLabel } from "@material-ui/core";
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import NavBar from '../NavBar/NavBar'; 
 import AddTikTokForm from '../AddTikTokForm/AddTikTokForm';
 import Home from '../Home/Home';
+import { getOembed } from '../../api-calls';
 import { Route, Switch } from "react-router";
 import { Alert } from '@material-ui/lab';
 
 const App = () => {
   const [initTikToks, setInitTikToks] = useState([
+    'https://www.tiktok.com/576582&is_from_webapp=v1&is_copy_url=0',
```

Comment:
Is this basically mock data for the initial load? Could you use a data file for this and import it, to clean up the bulky code?

Reply:
It is! I don't know why I didn't think of that earlier, especially considering I might want to remove it eventually